virttest/env_process.py

```diff
@@ -79,6 +80,8 @@
 postprocess_vm_on_hook = None
 postprocess_vm_off_hook = None
 
+#: Object to handle kvm module reloads with certain parameters
+_kvm_module_handler = None
```

Comment:
Top level global variables should be named UPPER_CASE

Reply:
Done.